app/uk/gov/hmrc/openidconnect/userinfo/domain/package.scala

```diff
@@ -21,31 +21,48 @@ import play.api.libs.json._
 import play.api.libs.functional.syntax._
 
 package object domain {
-  implicit val dateReads = Reads.jodaDateReads("yyyy-MM-dd")
-  implicit val dateWrites = Writes.jodaDateWrites("yyyy-MM-dd")
-  implicit val addressFmt = Json.format[Address]
-  implicit val userInfoFmt = Json.format[UserInfo]
-  implicit val apiAccessFmt = Json.format[APIAccess]
 
   implicit val desUserName : Reads[DesUserName] = (
-    (JsPath \ "firstForenameOrInitial").readNullable[String] and
-      (JsPath \ "secondForenameOrInitial").readNullable[String] and
-      (JsPath \ "surname").readNullable[String]
-    )(DesUserName.apply _)
+  (JsPath \ "firstForenameOrInitial").readNullable[String] and
+  (JsPath \ "secondForenameOrInitial").readNullable[String] and
+  (JsPath \ "surname").readNullable[String]
+  )(DesUserName.apply _)
 
   implicit val desAddress : Reads[DesAddress] = (
-    (JsPath \ "line1").readNullable[String] and
-      (JsPath \ "line2").readNullable[String] and
-      (JsPath \ "line3").readNullable[String] and
-      (JsPath \ "line4").readNullable[String] and
-      (JsPath \ "postcode").readNullable[String] and
-      (JsPath \ "countryCode").readNullable[Int]
-    )(DesAddress.apply _)
+  (JsPath \ "line1").readNullable[String] and
+  (JsPath \ "line2").readNullable[String] and
+  (JsPath \ "line3").readNullable[String] and
+  (JsPath \ "line4").readNullable[String] and
+  (JsPath \ "postcode").readNullable[String] and
+  (JsPath \ "countryCode").readNullable[Int]
+  )(DesAddress.apply _)
 
   implicit val desUserInfo : Reads[DesUserInfo] = (
-    (JsPath \ "names" \ "1").read[DesUserName] and
-      (JsPath \ "dateOfBirth").readNullable[LocalDate] and
-      (JsPath \ "addresses" \ "1").read[DesAddress]
-    )(DesUserInfo.apply _)
+  (JsPath \ "names" \ "1").read[DesUserName] and
+  (JsPath \ "dateOfBirth").readNullable[LocalDate] and
+  (JsPath \ "addresses" \ "1").read[DesAddress]
+  )(DesUserInfo.apply _)
+
+  implicit val idformat = Json.format[EnrolmentIdentifier]
+  implicit val format = Format(
+    ((__ \ "key").read[String] and
+      (__ \ "identifiers").read[Seq[EnrolmentIdentifier]] and
+      (__ \ "state").readNullable[String]) {
+      (key, ids, optState) =>
+        Enrolment(
+          key,
+          ids,
+          optState.getOrElse("Activate")
+        )
```

Comment:
To be honest I have no idea :( Although this part in `auth` has _Activate_ as the value. If you take a look at `Enrolment` class in `auth` you'll see that sometimes it's _Acitvate_, sometimes _Acitvated_ and sometimes _activated_